src/components/helpers.tsx

```diff
@@ -0,0 +1,16 @@
+import { css } from "styled-components"
+
+/**
+ * Helper function to display an element as a block that inherits its parents width
+ * @param margin value in pixels to remove from width 100%
+ */
+export const block = (margin: number = 0) => {
+    return props => {
+        if (props.block)
+            return css`
+                display: block;
+                width: calc(100% - ${margin}px);
+                margin: 10px auto;
+            `
+        }
```

Comment:
I agree and discussed this in Slack, but seems it got merged maybe before/during I raised that.

I had just assumed that tslint was going to enforce 2 spaces as  default, I can’t believe [it’s still not in core](https://github.com/palantir/tslint/issues/581), however that ticket does point out [this external set of rules](https://github.com/buzinas/tslint-eslint-rules/blob/master/src/docs/rules/terIndentRule.md) that do support it, so we should just switch to that.